recipes/imagemgagick/all/conanfile.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from conans import ConanFile, tools, AutoToolsBuildEnvironment, MSBuild
+import os
+import shutil
+import glob
+
+
+class ImageMagicConan(ConanFile):
+    name = "imagemagick"
+    description = ("ImageMagick is a free and open-source software suite for displaying, converting, and editing "
+                   "raster image and vector image files")
+    topics = ("conan", "imagemagick", "images", "manipulating")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ImageMagick/ImageMagick"
+    license = "ImageMagick"
+    exports = ["LICENSE.md"]
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "hdri": [True, False],
+               "quantum_depth": [8, 16, 32],
+               "zlib": [True, False],
+               "bzlib": [True, False],
+               "lzma": [True, False],
+               "lcms": [True, False],
+               "openexr": [True, False],
+               "jpeg": [True, False],
+               "openjp2": [True, False],
+               "png": [True, False],
+               "tiff": [True, False],
+               "webp": [True, False],
+               "xml": [True, False],
+               "freetype": [True, False],
```

Comment:
Please prepend these options with `with_`.
We've been doing this for a lot of recipes (not all though).
But it's nice to have some uniformity.